packages/headless/src/app/case-assist-engine/case-assist-engine.test.ts

```diff
@@ -25,6 +25,7 @@ describe('buildCaseAssistEngine', () => {
   beforeEach(() => {
     options = {
       configuration: getSampleCaseAssistEngineConfiguration(),
+      loggerOptions: {level: 'silent'},
```

Comment:
That can be quite annoying when debugging tests, no ?

Reply:
Hmm, the log here is intentional, created by the test on line 34 passing an invalid id (empty string).

To silence the error, we could `jest.mock` or pass this config option. There are several test suites using this approach to silence pino (try a grep on `silent`). A close comparison would be the [search engine](https://github.com/coveo/ui-kit/blob/master/packages/headless/src/app/search-engine/search-engine.test.ts#L21).

I feel I did not answer your concern though. I'm not sure what you have in mind when debugging tests?

Reply:
Ahhhhh nvm me. 
I was confused with Jest option to run "silent", but here it's Headless engine config.

I'm good with this then 👍